### DISCUSSION TITLE
Login error with permissions

### DISCUSSION THREAD

**Discussion by jjgonc** [directus/directus#7122] [Archive]
Hey, I'm developing a order system using Directus.
I was trying to set the permissions to only the admin app see all the orders and the users only check the orders they created, but I probably clicked a wrong permission and, as it saves automatically, now I might have some permissions wrong

I tried to get custom permissions in the accessibility of the collection I want to show only if the user logged has created and did the following: 
![Screenshot from 2021-08-02 10-13-23](https://user-images.githubusercontent.com/61986130/127873303-8e894ff7-68e4-457a-8a0c-dac924874d9e.png)
but this isn't working to show the way I want...

Can I get help on this?

**EDIT:** managed to solve the login problem

**Comment by rijkvanzanten** [comment]
It seems like you solved the problem already? 

If not, what are your permissions currently

**Reply by jjgonc** [reply]
The login problem I solved, but I can't get the permissions to work propperly as I'd like

So, let suppose this is for a car shop and I have the following roles: User (normal website user that can only see and place new orders), Funcionario (**Mechanic** in english that can see all the orders and edit their status (done, processing and done) and remove orders) and the admin role (all access)

I'd like to have those permisions set.

![Screenshot from 2021-08-02 15-05-38](https://user-images.githubusercontent.com/61986130/127875646-be4435b7-4c30-4e18-9420-8aebf277c5e2.png)

In the **user** and the **funcionario** (mechanic or worker of the place) I've set the permissions to "Recommended default"
![Screenshot from 2021-08-02 15-15-08](https://user-images.githubusercontent.com/61986130/127875794-45263c68-488d-469f-86c8-f519812f3df1.png)


Note: when I dont have an active login and I go to the **Public role**, If I dont have an active login, I can see and edit all the orders.

Thanks for the help once again!

**Reply by rijkvanzanten** [reply]
> I tried to get custom permissions in the accessibility of the collection I want to show only if the user logged has created and did the following:
> Screenshot from 2021-08-02 10-13-23
> but this isn't working to show the way I want...

This does sound like the right way to go to me. Does your collection have a `user_created` field?
